orchestra/tests/unit/specs/mistral/test_workflow_spec_validate.py

```diff
@@ -115,3 +115,61 @@ def test_empty_task_default_list(self):
         wf_spec = self.instantiate(wf_def)
 
         self.assertDictEqual(wf_spec.inspect(), {})
+
+    def test_type_direct(self):
+        wf_def = """
+            version: '2.0'
+
+            sequential:
+                description: A basic sequential workflow.
+                type: direct
+                tasks:
+                    task1:
+                        action: std.noop
+        """
+
+        wf_spec = self.instantiate(wf_def)
+
+        self.assertDictEqual(wf_spec.inspect(), {})
+
+    def test_type_reverse(self):
+        wf_def = """
+            version: '2.0'
+
+            sequential:
+                description: A basic sequential workflow.
+                type: reverse
+                tasks:
+                    task1:
+                        action: std.noop
+        """
+
+        wf_spec = self.instantiate(wf_def)
+
+        self.assertDictEqual(wf_spec.inspect(), {})
+
+    def test_type_junk_throws_error(self):
+        wf_def = """
```

Comment:
Can we move these workflows to fixtures?

Reply:
@m4dcoder asked me to move these back from fixtures into this file.

Reply:
@bigmstone Yes I did. If the WF definitions are shared, then let's move to fixtures.  Otherwise for definition that we used only once, I prefer them to be inlined so we don't go back and forth.